### PR TITLE
Fix bunch of GitHub integration issues faced by Vlang

### PR DESCRIPTION
**Pull request by akhismat**
1. Add min support for md comments to fix GitHub integration:

* Comments are parsed from GH issue description.
* Comments are never shown in Huly issues, but seen when comparing with GH version

2. Use original bullets and markers when importing issues from GH
3. Ignore changes in empty lines

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzA1MzExMjFmMmY4YWNlMzU0NDg2ZTciLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIn0.ovh38HObYlOGCjxk8efcfVGPG6v7NgsO0a_FLKk0gCU">Huly&reg;: <b>UBERF-8389</b></a></sub>

### CODE REVIEW

**Comment by haiodo** [comment]
Could we have few tests [Anna Khismatullina](https://front.hc.engineering/browse/?workspace=platform&_class=contact%3Aclass%3APerson&_id=66c82dfe93788b742197a18e&label=Anna%20Khismatullina) ?

File: packages/text/src/markdown/parser.ts [hunk 1]
```diff
@@ -619,6 +620,17 @@ export class MarkdownParser {
     return doc
   }
 
+  htmlCommentRule: RuleCore = (state: StateCore): boolean => {
```
Comment (haiodo) [inline packages/text/src/markdown/parser.ts:648]: May be we as well add any other html rules same way?

Comment (akhismat) [reply packages/text/src/markdown/parser.ts:648]: Sure, filed a task for this https://front.hc.engineering/workbench/platform/tracker/UBERF-8402
Will do it right after the clickup import

File: packages/text/src/nodes/comment.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  parseHTML () {
+    return [
+      {
+        tag: 'comment'
+      }
+    ]
+  },
+
+  renderHTML () {
+    return ['']
```
Comment (aonnikov) [inline packages/text/src/nodes/comment.ts]: Let's do some rendering here, otherwise content will be lost after copying and pasting.

Comment (akhismat) [reply packages/text/src/nodes/comment.ts]: Removed this code as not used

**Review by aonnikov — APPROVED** [review]

**Review by haiodo — APPROVED** [review]

**Pull request merged**